proto/prysm/v1alpha1/BUILD.bazel

```diff
@@ -65,8 +65,8 @@ ssz_gen_marshal(
         "BeaconBlockBodyCapella",
         "SignedBeaconBlockCapella",
         "BeaconBlockDeneb",
-	"BeaconBlockePBS",
-	"SignedBeaconBlockePBS",
+	    "BeaconBlockEpbs",
+	    "SignedBeaconBlockEpbs",
```

Comment:
better to remove the extra indentation, Preston typically complains about this

Reply:
weird, this actually fixes it to align with the rest, but before it was off : )
![Screenshot 2024-04-24 at 11 17 02 AM](https://github.com/prysmaticlabs/prysm/assets/21316537/0178fe21-3636-4629-8a5d-67c0c95f19a8)

Reply:
This is how engine build.bazel looks to me 
![Screenshot 2024-04-24 at 11 18 00 AM](https://github.com/prysmaticlabs/prysm/assets/21316537/c1be2bac-1a8a-4417-8720-2cad40f7f1d9)